Dockerfile

```diff
@@ -1,26 +1,34 @@
+# Base stage
 FROM node:18-alpine AS base
 
 ENV CHROME_BIN="/usr/bin/chromium-browser"
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD="true"
 
 WORKDIR /usr/src/app
 
-RUN \
-  apk --no-cache upgrade && \
-  apk add --no-cache udev ttf-opensans unifont chromium ca-certificates dumb-init && \
-  rm -rf /tmp/*
+RUN apk --no-cache upgrade && \
+    apk add --no-cache udev ttf-opensans unifont chromium ca-certificates dumb-init && \
+    rm -rf /tmp/*
 
-FROM base as build
-
-COPY . ./
+# Development dependencies stage
+FROM base AS dev-dependencies
 
+COPY package.json yarn.lock ./
 RUN yarn install --pure-lockfile
+
+# Build stage
+FROM dev-dependencies AS build
+
+COPY . ./
 RUN yarn run build
```

Comment:
```suggestion
# Build stage
FROM base AS build

COPY . ./

RUN yarn install --pure-lockfile
RUN yarn run build
```

Reply:
@McTonderski Could you take a look at this suggestion, please? It simplifies a bit the code while still keeping the dev dependencies out of the final image

Reply:
hey, Thank you for pushing that MR !